layers/+source-control/git/README.org

```diff
@@ -152,7 +152,7 @@ Here are the often used bindings inside a =status buffer=:
 | ~n~         | next search occurrence                              |
 | ~N~         | previous search occurrence                          |
 | ~o~         | revert item at point                                |
-| ~P P~       | push                                                |
+| ~P~         | Open a =push buffer=                                |
```

Comment:
Same comment here, this line is about `pushing` not just `open a push buffer`.
According to new Magit it is now `P u` and we should adapt the description to `push to tracked branch`.